anthem/lyrics/loaders.py

```diff
@@ -57,7 +57,7 @@ def read_csv(data, dialect='excel', encoding='utf-8', **fmtparams):
 
 def load_rows(ctx, model, header, rows):
     if isinstance(model, basestring):
-        model = ctx.env[model]
+        model = ctx.env[model].with_context(tracking_disable=True)
```

Comment:
And in the case there a need to enable the tracking, is there a solution to force the value when calling `load_rows`?

Reply:
Yes, by passing an instance of `models` with the context key defined. I added a commit to cover more use cases.

Reply:
@sebalix prior to this change the only way to define the context was already to pass a RecordSet with a specific context, so for me the current change makes sense.